docker/Dockerfile

```diff
@@ -28,10 +28,7 @@ RUN export GOOS=$TARGETOS && \
 
 FROM $BASEIMAGE
 COPY --from=builder /go/src/sigs.k8s.io/secrets-store-csi-driver/_output/secrets-store-csi /secrets-store-csi
-# upgrading zlib1g due to CVE-2018-25032
-# upgrading gzip and liblzma5 due to CVE-2022-1271
-# upgrading libssl1.1 due to CVE-2022-1292
-RUN clean-install ca-certificates mount zlib1g gzip liblzma5 libssl1.1
+RUN clean-install ca-certificates mount
```

Comment:
are these CVEs fixed in v1.3.0?

Reply:
yeah, we have image scan as part of every PR: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_secrets-store-csi-driver/959/pull-secrets-store-csi-driver-image-scan/1529591015915130880